pycsob/client.py

```diff
@@ -102,14 +102,14 @@ def payment_init(self, order_no, total_amount, return_url, description, merchant
         :return: response from gateway as OrderedDict
         """
 
-        if len(description) > 255:
-            raise ValueError('Description length is over 255 chars')
+        if len(description) > 20:
```

Comment:
Parameter `description` is now used only to create "cart" data, so I changed the validation accordingly.